lib/samples/query_feature_table/README.md

```diff
@@ -0,0 +1,35 @@
+# Query feature table
+
+Find features in a feature table which match an SQL query.
+
+![Image of query feature table](query_feature_table.png)
+
+## Use case
+
+Query expressions can be used in ArcGIS to select a subset of features from a feature table. This is most useful in large or complicated data sets. A possible use case might be on a feature table marking the location of street furniture through a city. A user may wish to query by a TYPE column to return "benches". In this sample, we query a U.S. state by STATE_NAME from a feature table containing all U.S. states.
+
+## How to use the sample
+
+Input the name of a U.S. state into the text field. When you click "search", a query is performed and the matching features are highlighted or an error is returned.
```

Comment:
```suggestion
Input the name of a U.S. state into the text field. When you tap "search", a query is performed and the matching features are highlighted or an error is returned.
```
One to look out for where the design was written for desktop, we like to use "tap" for mobile devices.